stock_move_location/wizard/stock_move_location.py

```diff
@@ -207,9 +209,13 @@ def _get_move_values(self, picking, lines):
     def _create_move(self, picking, lines):
         self.ensure_one()
         move = self.env["stock.move"].create(self._get_move_values(picking, lines))
-        if not self.env.context.get("planned"):
-            for line in lines:
-                line.create_move_lines(picking, move)
+        for line in lines:
+            line.create_move_lines(picking, move)
```

Comment:
`create_move_lines()` works on a recordset.

```suggestion
        lines.create_move_lines(picking, move)
```